frontend/src/components/new-car-card/styles.module.scss

```diff
@@ -81,7 +81,9 @@
 }
 
 .carDescription {
+  max-height: 5.3rem;
   margin: 0;
+  overflow: hidden;
```

Comment:
Why should we just hide the overflown text? Because it could be confusing. But instead, we should use should show `...` as there's some more text available. There's a CSS prop `line-clamp` which makes it pretty easy